src/adr/references/documenting-architecture-decisions.md

```diff
@@ -0,0 +1,160 @@
+# [Documenting Architecture Decisions](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions)
```

Comment:
Licence? Or maybe we can simply link to archive.org if we are afraid of this article to be lost? https://web.archive.org/web/20190824074401/http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions